episodes/02-filedir.md

```diff
@@ -405,6 +405,35 @@ Note that if a directory named `Desktop` does not exist in your current working
 this command will return an error. Typically, a `Desktop` directory exists in your
 home directory, which we assume is the current working directory of your bash shell.
 
+:::::::::::::: callout
+
+### OneDrive Variations
+
+If a Windows machine is backed up with OneDrive, your Desktop may
+  be rendered from a folder within OneDrive, which will not match the contents of `~/Desktop`.
+  The OneDrive desktop should be accessible using one of the following commands
+  (if the name of the enterprise/org isn't clear, look through the output of `ls` to find
+  the right folder):
+  
+  ```bash
+  $ cd "~/OneDrive - Name Of Enterprise/Desktop"
+  $ cd "C:/Users/Username/OneDrive - Name Of Enterprise/Desktop"
+  ```
```

Comment:
```suggestion
  Run the command `ls` and examine the output, looking for a folder that has "OneDrive" in the name and note that folder name. It will likely include the name of your organization in it.  It may be something like `OneDrive - University of Example`
  Going forward, whenever the lesson or instructor uses the `Desktop` folder, you will need to include the OneDrive folder as well. For example if the instructor types...

``` bash
ls -F Desktop
```

You will need to type:

```bash
ls -F "OneDrive - Name Of Enterprise/Desktop"
```

Reply:
If the language in the above looks okay, I'll fix it not in a github comment.  It isn't interpreting my suggestion correctly